plugins/org.jboss.tools.docker.reddeer/src/org/jboss/tools/docker/reddeer/core/ui/wizards/NewDockerConnectionPage.java

```diff
@@ -88,24 +87,23 @@ public void setTcpUri(String uri) {
 		setConnectionName(uri);
 		new CheckBox("Use custom connection settings:").toggle(true);
 		new LabeledText("Location:").setText("");
-		new RadioButton("TCP Connection").click();
+		new RadioButton("TCP Connection").toggle(true);
 		new LabeledText("URI:").setText(uri);
 	}
 
 	public void pingConnection() {
 		Button testConnectionButton = new PushButton("Test Connection");
 		testConnectionButton.click();
 		new WaitUntil(new ShellWithTextIsAvailable("Success"));
-		Button okButton = new PushButton("OK");
-		okButton.click();
+		new PushButton("OK").click();
```

Comment:
Just a note. RedDeer has "shortcut" for this - OkButton (as well as CancelButton, FinishButton, NextButton, YesButton etc.)